packages/babel-traverse/test/scope.js

```diff
@@ -1127,4 +1127,21 @@ describe("scope", () => {
       `);
     });
   });
+
+  describe("constantViolations", () => {
+    it("var redeclarations should not be treated as constantViolations", () => {
+      const program = getPath(`
+        function v() { }
+        const G2 = () => {
+          var a = 1;
+          var {
+            currentPoint: a,
+            centp: v,
+          } = {};
+        }
+      `);
+      const a = program.scope.getBinding("v");
```

Comment:
```suggestion
      const bindingV = program.scope.getBinding("v");
```
since we already have `a` in the input source code. Can we also add an assertion for the `bindingA`?